### DISCUSSION TITLE
如何为生成的 vs 工程添加非编译文件

### DISCUSSION THREAD

**Discussion by Ariaszzzhc** [xmake-io/xmake#4668] [Q&A] [answered]
在 vs 中如果要开启 editorconfig 的功能，需要在工程中添加 .editorconfig 文件。

xmake 通过 *xmake project -k vsxmake* 中默认只有 target 包含的文件。

目前无法直接在 vs 工程中添加，因为 xmake 重新生成时会覆盖。

那如何在 xmake.lua 中配置呢？

**Comment by waruqi** [comment, marked as answer]
如果只是 vs 中添加显示和编辑，可以走 add_extrafiles
